util/http/http_client.h

```diff
@@ -128,5 +133,34 @@ template <typename Req> std::error_code Client::Send(const Req& req) {
   return HandleError(ec);
 }
 
+///////////////////////////////////////////////////////////////////////////////
+
+class SecureClient : public Client {
+ public:
+  // see https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_security_level.html
+  enum SecurityLevel {
```

Comment:
I prefer avoiding to duplicate APIs for this. If sane defaults work then we do not need it.
if they do not work then a caller that receives SSL_CTX object cal call whatever openssl function he desires.

Reply:
OK, I will remove it